manifests/init.pp

```diff
@@ -102,18 +102,19 @@
     ssl_cert_db            => $::certs::nss_db_dir,
     ssl_cert_password_file => $::certs::qpid::nss_db_password_file,
     ssl_cert_name          => 'broker',
+    interface              => 'lo',
```

Comment:
It's used. 

https://github.com/Katello/puppet-qpid/blob/master/manifests/init.pp#L18
https://github.com/Katello/puppet-qpid/blob/master/templates/qpidd.conf.erb#L42